src/api/routes/plugin.rs

```diff
@@ -31,7 +31,13 @@ mod tests {
     use actix_web::{test, App, HttpResponse};
```

Comment:
In order to have sdk support we should document plugins route in src/api/routes/docs so openapi spec includes it.

Reply:
Probably not part of this PR but mentioning just in case as it is something that is needed.